opm/grid/CpGrid.hpp

```diff
@@ -399,6 +399,8 @@ namespace Dune
         /// from whence the current CpGrid was constructed.
         const std::vector<int>& globalCell() const;
 
+        std::vector<std::shared_ptr<Dune::cpgrid::CpGridData>> chooseData() const;
```

Comment:
Would be cool, if we would prevent a copying a vector here.

Reply:
And some doxygen documentation would be highly appreciated. It might save others some time later.